End-to-End-Tests.md

```diff
@@ -299,17 +428,17 @@ then you can look for the report that includes this message. Times like `10003ms
 
 Sometimes you'll get screenshots that just aren't very helpful. For example, a lot of screenshots show the login page for some reason. You can check other examples of a flake though. One of the others might have a useful screenshot.
 
-### Videos of End-to-end Tests
+### Downloading screen recordings
 
-The Automated QA team has implemented a video recording system -- now, videos of end-to-end test runs on GitHub Actions may be accessed. This system will help developers solve problems in e2e tests that only occur on CI or are difficult to replicate locally.
+When screen recordings are enabled, we capture video of the test running on GitHub Actions. This helps developers solve problems in E2E tests that only occur on CI or are difficult to replicate locally.
```

Comment:
Link this to an explanation for how to enable?

Reply:
I clarified the later paragraph that describes when recordings are generated

Reply:
Yup, but what I'm trying to point out is that the main thing developers are going to look for when reading this section is "how do I get screen recordings for my currently-failing test". I think it might be worth having that really clear and front-and-center, rather than just a description of what the system currently is.

More generally, you might want a step-by-step list somewhere of "these are the steps you should take when debugging an e2e test" (one of which presumably involves getting screen recordings).

Reply:
Ahh, that makes sense. done